src/Streamly/LZ4.hs

```diff
@@ -66,32 +65,37 @@ import Streamly.Internal.LZ4
 -- Compression
 --------------------------------------------------------------------------------
 
--- | @compressChunks config speedup stream@ compresses an input stream of @Array
--- word8@ using the configuration defined by @config@. The resulting stream is
--- of type @Array Word8@ where each array represents a compressed input
--- block. @speedup@ is a compression speedup factor, the more the value of
--- @speedup@ the faster the compression but the size of compressed data may
--- increase. The factor should be between 1 and 65537 inclusive, if it is less
--- than 1 it is set to 1 if it is more than 65537 then it is set to 65537.
+-- XXX Make speedup part of the BlockConfig?
```

Comment:
If something is not used by both, the current and its dual, I don't think the config is a good place for it. `speedup` is not used by both, `compress` and `decompress`.

Reply:
Fair point, I will edit the comment.